helix-view/src/document.rs

```diff
@@ -1331,6 +1414,11 @@ impl Document {
         &self.text
     }
 
+    #[inline]
+    pub fn full_range(&self) -> Range {
+        Range::new(0, self.text.len_chars())
+    }
+
```

Comment:
I dont think this needs a dedicated helper method. We have far more common operations that don't have helper methods. Adding too many helper methods clutters the (already very large) document impls even more